class CustodianError(Exception):
    pass


class BadRelatesTo(Exception):
    pass
